GunplaOnlineShop/GunplaOnlineShop/Data/Migrations/20200604025414_tryagain.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace GunplaOnlineShop.Data.Migrations
+{
+    public partial class tryagain : Migration
```

Comment:
empty migration file?  Up and Down methods are empty, delete this migration file